### PR TITLE
TSLint cleanups

### PR DESCRIPTION
**Pull request by jmyersmsft**
@zarenner @phil-hodgson

Not 100% tslint-clean, but close (in particular I haven't decided whether to disable the rule about not allowing property access via string literals, -- foo["bar"] vs foo.bar)

### CODE REVIEW

File: Tasks/Common/nuget-task-common/LocationApi.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import Q = require('q');
-import vstsWebApi = require('vso-node-api/WebApi');
-import vstsClientBases = require('vso-node-api/ClientApiBases');
-import VsoBaseInterfaces = require('vso-node-api/interfaces/common/VsoBaseInterfaces')
+import * as Q from "q";
```
Comment (phil-hodgson) [inline Tasks/Common/nuget-task-common/LocationApi.ts]: I haven't seen this syntax before. What does it do?

Comment (jmyersmsft) [inline Tasks/Common/nuget-task-common/LocationApi.ts]: Same as what it's replacing, just the newer style. Compiles down to the same JS code

File: Tasks/Common/nuget-task-common/NuGetQuirks.ts [hunk 2]
```diff
@@ -70,33 +69,33 @@ const allQuirks: QuirkDescriptor[] = [
         // but it would be less than 1707
         versionRanges: [
             halfOpenRange(nuget320, nuget350_1707),
-            halfOpenRange(nuget351, nuget351_1707)]
+            halfOpenRange(nuget351, nuget351_1707)],
```
Comment (phil-hodgson) [inline Tasks/Common/nuget-task-common/NuGetQuirks.ts]: tslint wants you to add the useless extra comma?

**Comment by phil-hodgson** [comment]
:shipit:

**Pull request merged**